src/main/scala/nl.knaw.dans.easy.deposit/DepositDir.scala

```diff
@@ -197,7 +197,7 @@ case class DepositDir private(baseDir: File, user: String, id: UUID) extends Deb
     if (doi == dm.doi) Success(())
     else {
       logger.error(s"DOI in datasetmetadata.json [${ dm.doi }] does not equal DOI in deposit.properties [$doi]")
-      Failure(InvalidDoiException(id))
+      Failure(new InvalidDoiException(id))
```

Comment:
IntelliJ tells me `new` is not necessary here... also in other places in this PR.